viz/tap/api/main.go

```diff
@@ -28,6 +29,9 @@ func Main(args []string) {
 	trustDomain := cmd.String("identity-trust-domain", defaultDomain, "configures the name suffix used for identities")
 	enablePprof := cmd.Bool("enable-pprof", false, "Enable pprof endpoints on the admin server")
 
+	var tapIgnoredHeaders = &stringMap{}
+	cmd.Var(tapIgnoredHeaders, "tap-ignored-headers", "list of headers to ignore")
```

Comment:
Might be simpler to just use `cmd.StringSliceVar`

Reply:
A comma-separated list was my original implementation but I took your feedback from the previous comment:
>  I would however have that be a yaml array instead of having to parse a string

Maybe cause that was originally in JS, let me know which you prefer :)

Reply:
Re: using `cmd.StringSliceVar` instead:

I intentionally implemented it as a string map instead of a list to get O(1) on the search

Reply:
I wouldn't consider performance in this particular case, but you're right about `flag` by itself not offering something out of the box for handling slices :+1:

Reply:
... so I'm good with what you implemented here :+1: